pkgs/desktops/rox/rox-filer/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, fetchurl, autoconf, pkgconfig, gtk, libSM, shared_mime_info,
+  libxml2, libxslt, docbook_xml_dtd_412 }:
+
+stdenv.mkDerivation rec {
+  name = "rox-filer-2.11";
+
+  src = fetchurl {
+    url = "https://github.com/rox-desktop/rox-filer/tarball/2b9737f13900d68a2726cab6812f950157345345";
+    sha256 = "0bz014c75c38jq6lca0ccz9w6cfbi0hx2pbfm6by00p9ys1ma504";
+  };
+
+  buildInputs = [ pkgconfig gtk autoconf libSM libxml2 shared_mime_info ];
+
+  unpackCmd = "tar -xvzf $src";
+
+  buildPhase = ''
+    ./ROX-Filer/AppRun --compile
+
+    substituteInPlace ROX-Filer/src/Docs/Makefile \
+      --replace xmllint ${libxml2.bin}/bin/xmllint \
+      --replace xsltproc ${libxslt.bin}/bin/xsltproc
+
+    for xmlFile in ROX-Filer/src/Docs/Manual*.xml; do
+      substituteInPlace  "$xmlFile" \
+        --replace "/usr/share/sgml/docbook/dtd/xml/4.1.2/docbookx.dtd" \
+                  "${docbook_xml_dtd_412}/xml/dtd/docbook/docbookx.dtd"
+    done
+
+    (cd ROX-Filer/src/Docs && make)
+  '';
+
+  installPhase = ''
+    mkdir -p $out/{bin,share/ROX-Filer,share/Choices/MIME-types,share/man/man1}
+    cp -rp ROX-Filer/{Help,Messages,ROX,images} $out/share/ROX-Filer
+    cp -p rox.1 $out/share/man/man1
+    cp -p ROX-Filer/{AppInfo.xml,AppRun,Options.xml,ROX-Filer,Templates.ui,subclasses,style.css,.DirIcon} \
+          $out/share/ROX-Filer
+    cp -p Choices/MIME-types/* $out/share/Choices/MIME-types
+
+    echo '#!/bin/sh' > $out/bin/rox
+    echo "exec $out/share/ROX-Filer/AppRun \"\$@\"" >> $out/bin/rox
+    chmod 755 $out/bin/rox
+    echo "out $out"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A RISC OS-like filer for X, the file manager at the core of the ROX desktop";
+    homepage = http://rox.sourceforge.net/desktop/ROX-Filer;
+    platforms = stdenv.lib.platforms.linux;
```

Comment:
since `stdenv.lib` is already in context with the `with stdenv.lib` above you can just address `platforms.linux`.